internal/payloads/permissions_payload.go

```diff
@@ -0,0 +1,28 @@
+package payloads
+
+import (
+	"net/http"
+
+	"github.com/go-chi/render"
+)
+
+type PermissionsResponse struct {
+	Valid           bool
+	MissingPolicies []string
```

Comment:
The only thing is this name - I am not sure if on GCP these will be policies. I suggest to rename this to MissingEntities or just Errors where you put strings like "missing policy XXX", "missing policy YYY" or something like that.

For example in Azure it is called Roles and Permissions.